recipes/configs/llama3_2_vision/90B_full.yaml

```diff
@@ -1,17 +1,17 @@
-# Config for single device full finetuning in full_finetune_single_device.py
+# Config for multi-device full finetuning in full_finetune_distributed.py
 # using a Llama3.2 90B Vision Instruct model
 #
 # This config assumes that you've run the following command before launching:
 #   tune download meta-llama/Llama-3.2-90B-Vision-Instruct --output-dir /tmp/Llama-3.2-90B-Vision-Instruct --ignore-patterns "original/consolidated*"
 #
-# To launch on a single device, run the following command from root:
+# To launch on 8 devices, run the following command from root:
 #   tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full
 #
 # You can add specific overrides through the command line. For example
 # to override the checkpointer directory while launching training:
-#    tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#   tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
 #
-# This config works best when the model is being fine-tuned on 2+ GPUs.
+# This config needs 8 GPUs to run.
```

Comment:
8 A100s?

Reply:
do we specify the GPU in any of the config comments...?

Reply:
tru.....